src/plugin-store.svelte

```diff
@@ -97,49 +99,47 @@ function toggleOfficialPlugin(plugin: Plugin, isEnabled: boolean) {
 let externalPlugins: Plugin[] = [];
 
 async function fetchExternalPlugins() {
-	const externalPluginsMockup: Plugin[] = [
-		{
-			name: "Type Designer",
-			author: "SprintEins",
-			src: "https://sprinteins.github.io/oscd-plugins/type-designer/index.js",
-			kind: "editor",
-			installed: false,
-		},
-		{
-			name: "Auto Doc",
-			author: "SprintEins",
-			src: "https://sprinteins.github.io/osdc-plugins/auto-doc/index.js",
-			kind: "editor",
-			installed: false,
-		},
-	];
+	const url = "https://sprinteins.github.io/oscd-plugin-store/plugins.json";
 
-	externalPlugins = externalPluginsMockup;
+	const result = await fetch(url);
+	const data = await result.json();
+	externalPlugins = data.plugins;
 }
 
 fetchExternalPlugins();
 
 let showOnlyInstalled = false;
 let searchFilter = "";
 
-function combineAllPlugins(local: Plugin[]): Plugin[] {
+function combineAllPlugins(local: Plugin[], external: Plugin[]): Plugin[] {
 	const plugins: Plugin[] = [...local];
 
 	for (let i = 0; i < externalPlugins.length; i++) {
-		if (!localPlugins.some((it) => it.name === externalPlugins[i].name)) {
-			plugins.push(externalPlugins[i]);
+		if (!localPlugins.some((it) => it.name === external[i].name)) {
+			plugins.push(external[i]);
 		}
 	}
 
+	plugins.sort((a, b) => {
+		return a.name.localeCompare(b.name);
+	});
+
```

Comment:
How about we first sort by author then plugin name?
One would probably would like to see all the plugins from one author after each other